spec/rubocop/cop/style/comment_annotation_spec.rb

```diff
@@ -113,12 +113,27 @@
     RUBY
   end
 
-  it 'accepts a keyword that is somewhere in a sentence' do
-    expect_no_offenses(<<-RUBY.strip_indent)
-      # Example: There are three reviews, with ranks 1, 2, and 3. A new
-      # review is saved with rank 2. The two reviews that originally had
-      # ranks 2 and 3 will have their ranks increased to 3 and 4.
-    RUBY
+  context 'comments in consecutive lines' do
```

Comment:
I'm not certain that we should bucket a "multi-line" comment and consecutive inline comments together.

Reply:
I regard multi-line comments containing keywords as an exception from the rule, and consecutive inline comments as exception from the exception. That's also how I implemented the fix. My intention bucketing them together was to show this relationship, but I wasn't certain, too.

Reply:
I'd rather just slit the context to avoid the potential confusion.